website/docs/workloads/mlperf/mlperf-developer-guide.md

```diff
@@ -0,0 +1,221 @@
+| N/A   31C    P0             52W /  400W |       1MiB /  40960MiB |      0%      Default |
+|                                         |                        |             Disabled |
++-----------------------------------------+------------------------+----------------------+
+|   7  NVIDIA A100-SXM4-40GB          Off |   0000000E:00:00.0 Off |                    0 |
+| N/A   30C    P0             53W /  400W |       1MiB /  40960MiB |      0%      Default |
+|                                         |                        |             Disabled |
++-----------------------------------------+------------------------+----------------------+
+
++-----------------------------------------------------------------------------------------+
+| Processes:                                                                              |
+|  GPU   GI   CI        PID   Type   Process name                              GPU Memory |
+|        ID   ID                                                               Usage      |
+|=========================================================================================|
+|  No running processes found                                                             |
++-----------------------------------------------------------------------------------------+
+```
+- **Docker**: A platform to use containers. MLPerf inference will run benchmarks within a docker container.  
+Docker is installed with DockerInstallation.
+```
+{
+    "Type": "DockerInstallation",
+    "Parameters": {
+        "Scenario": "InstallDocker"
+    }
+}
+```
+- **NVIDIA Container Toolkit**: A set of tools which enable the use of NVIDIA GPUs within docker containers.  
+Nvidia container toolkit is installed with NvidiaContainerToolkitInstallation.
+```
+{
+    "Type": "NvidiaContainerToolkitInstallation",
+    "Parameters": {
+        "Scenario": "InstallNvidiaContainerToolkit"
+    }
+}
+```
+
+## Running a Benchmark
+There are a few setup steps before running the benchmark:
+- **make prebuild**: Download the docker container image and launch the container.
+The remaining commands are run within the container. In order to avoid launching the docker container shell,
+the file is replaced with Virtual Client. The [replacing Makefile.docker file](https://github.com/microsoft/VirtualClient/blob/main/src/VirtualClient/VirtualClient.Actions/MLPerf/Makefile.docker)
+does not launch the docker container shell.
+- **make download_data BENCHMARKS="bert"**: Download datasets necessary to run the benchmark.
+- **make download_model BENCHMARKS="bert"**: Download pre-trained model to be tested.
+- **make preprocess_data BENCHMARKS="bert"**: Formats data to be used in the benchmark.
+- **make build**: Compile and build the executable to run the benchmark.
+
+To actually run the benchmark:
+- **make run RUN_ARGS='--benchmarks=bert --scenarios=Offline,Server,SingleStream --config_ver=default --test_mode=PerformanceOnly --fast**: Run performance mode which focuses
```

Comment:
Would be great to mention what each of these arg mean or the link to the doc to explain that.

Reply:
I stated what each argument is now. Most of these are explained a little more throughout the developer guide.